src/js/reducers/approversReducer.jsx

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash';
+
+import { FETCH_APPROVERS } from 'actions/types';
+
+const initialState = {
+  data: [],
+  fetched: false,
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case FETCH_APPROVERS:
+      if (action.payload !== undefined) {
```

Comment:
why are we strict to `=== undefined` instead of checking the truthiness?

Reply:
This is something I also noticed in other reducers, but I am not sure what is it trying to solve.
When removing the `!== undefined` everything seems to work fine but maybe I am not aware of some case which this `if()` fixes.
It should be safe to remove it I guess